perfkitbenchmarker/packages/fio.py

```diff
@@ -147,7 +147,13 @@ def ParseResults(job_file, fio_json_result):
         lat_metadata = {
             'min': job[mode]['clat']['min'],
             'max': job[mode]['clat']['max'],
-            'stddev': job[mode]['clat']['stddev']}
+            'stddev': job[mode]['clat']['stddev'],
+            '1th': job[mode]['clat']['percentile']['1.000000'],
```

Comment:
`1th` should be `1st`.

Other benchmarks use p1, p5, p50, etc. Maybe we could standardize?